prompt_prototype.rb

```diff
@@ -41,11 +41,23 @@ def morph_records(scraper, query)
   JSON.parse(RestClient.get(url), symbolize_names: true).map { |d| [d[:id], d]  }.to_h
 end
 
-wikidata_records = wikidata_sparql("SELECT ?item ?itemLabel WHERE { ?item wdt:P39 wd:#{wikidata_membership_item}.  SERVICE wikibase:label { bd:serviceParam wikibase:language \"[AUTO_LANGUAGE],en\". } }")
-morph_records = morph_records(morph_scraper, scraper_sql)
-
 country_slug, house_slug = ep_country_and_house.split('/')
 popolo = Everypolitician::Index.new.country(country_slug).legislature(house_slug).popolo
+
+house_wikidata_id = popolo.organizations.find { |o| o.classification == 'legislature' }.wikidata
```

Comment:
`.find_by(classification: 'legislature')` would also be better than rolling your own `.find`, but yes, ideally you wouldn't need to do either.